content/blog/20240329_fluent-package-v5.0.3-has-been-released.md

```diff
@@ -0,0 +1,51 @@
+# fluent-package v5.0.3 has been released
+
+Hi users!
+
+We have released fluent-package [v5.0.3](https://github.com/fluent/fluent-package-builder/releases/tag/v5.0.3)
+fluent-package is a stable distribution package of Fluentd. (successor of td-agent v4)
```

Comment:
This becomes one line, so we should add `.` or insert an empty line.

![Uploading Screenshot from 2024-03-29 14-31-47.png…]()

Reply:
Thanks!
I've fixed it.